developer-docs/how-to-guide/generate_apikey.md

```diff
@@ -160,19 +160,20 @@ The remaining fields of the JWT can be filled based on the details of the reques
           "msgid": "c3049b36249a3c9f8891cb1999777743c"
        }
     }
-    
+ ```   
 The generated JWT from the request: 
 
-    javascript
+```  
     eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJhMzZjMzA0OWIzNjI0OWEzYzlmODg5MWNiMTI3MjQzYyIsImlhdCI6IjE0NDI0MjYyMzE2MDAiLCJpZCI6Im9yZy5la3N0ZXAuYXBpLmRhdGFTZXQiLCJxc2giOiI4MDYzZmY0Y2ExZTQxZGYwZjYyMDdkNDkxY2Y2ZGFkN2M2NmVhNzk3YjQ2MTRiNyIsInZlcnNpb24iOiIxLjAiLCJwYXJhbXMiOnsiZGlkIjoiZmYzMDVkNTQtODViNC0zNDFiLWRhMmYtZWI2YjllNTQ2MGZhIiwibXNnaWQiOiJjMzA0OWIzNjI0OWEzYzlmODg5MWNiMTk5OTc3Nzc0M2MifX0.rww4BN-EfnAXKTRltQChYA3KM31GzyEF-YpYvIBzw1I
+```
 
 Usually, the JWT will be created programmatically by your application. JWT libraries exist for all popular programming languages. However, you may also use the web-based tool - [jwtbuilder](http://jwtbuilder.jamiekurtz.com/) to generate the JWT. Note that when using this tool, the **key** should be set as the **iss** field in the claims section of the payload and the **secret** obtained should be set as the **Key** field in the Signed JWT section. The Key will be used to generate a signed JWT.
```

Comment:
@seventunes pls use {:target="_blank"} with url.. Otherwise, this page will open in the same tab rather than opening in new tab. Read this - https://stackoverflow.com/questions/3492153/markdown-open-a-new-window-link